pkgs/development/python-modules/astroid/default.nix

```
@@ -45,11 +41,7 @@ buildPythonPackage rec {
 
   checkInputs = [
     pytestCheckHook
-  ];
-
-  disabledTests = [
-    # AssertionError: Lists differ: ['ABC[16 chars]yBase', 'Final', 'Generic', 'MyProtocol', 'Protocol', 'object'] != ['ABC[16 chars]yBase', 'Final', 'Generic', 'MyProtocol', 'object']
-    "test_mro_typing_extensions"
+    typing-extensions # see https://github.com/PyCQA/astroid/issues/1585
```

```suggestion
    typing-extensions
```
already included in a commit https://github.com/PyCQA/astroid/commit/4e9225e7191fcb2a92eea35ce05e17d0660a8c6b

---

Fixed.